prac_05/word_occurrences.py

```diff
@@ -0,0 +1,32 @@
+"""
+Word Occurrences
+Estimate: 40 minutes
+Actual: 45 minutes
+"""
+
+
+def main():
+    """Program that counts the occurrences of words in a string."""
+    # text = "This is a collection of words of nice words this is a fun thing it is"
+    get_words = input("Text: ")
+    count_words(get_words)
+
+
+def count_words(text):
+    """Converts words to lowercase and counts them."""
+    word_to_count = {}
+    words = text.split()
+    for word in words:
+        word = word.lower()
+        word_to_count[word] = word_to_count.get(word, 0) + 1
+    calculate_longest_word(word_to_count)
```

Comment:
Should you be calling a function in a function?

Reply:
@JoshuaDelosSantos - There's nothing wrong with calling a function in a function.